### DISCUSSION TITLE
Typo in Webhook Docs

### DISCUSSION THREAD

**Discussion by satc0ride** [community/community#19935] [Projects and Issues]
Hi, there are an typo on

https://docs.github.com/es/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-object-common-properties

seen in spanish, Propiedades appears like Propuidades

Regards

**Comment by baysideengineer** [comment]
Hi @satc0ride, in fact that all GitHub documentation is open source you can contribute and correct the typos if you want. Just fork the [repository](https://github.com/github/docs/edit/main/content/developers/webhooks-and-events/webhooks/webhook-events-and-payloads.md) make your changes and submit your pull request. We love to see you contributing to GitHub. Have a nice day.
